### ISSUE TITLE
[BUG 0.79.0] UI does not display file/model contents

### ISSUE THREAD

**Issue by martsec** [TobikoData/sqlmesh#2309]
Hi

I've just installed SQLMesh and it turns out the UI does not render any file. Since it's happening with all the browsers and the API is providing the correct file, I assume it might be a bug related to the UI itself. 

Python: 3.9.2
SLQMesh version: 0.79.0
Browsers: Latest Firefox, Chrome and Edge

## Steps to reproduce

Just the ones from https://sqlmesh.readthedocs.io/en/stable/quickstart/ui/#setup
```bash
pip install "sqlmesh"
pip install "sqlmesh[web,dev]"
mkdir sqlmesh-example
cd sqlmesh-example
sqlmesh init duckdb
sqlmesh ui
```

![imatge](https://github.com/TobikoData/sqlmesh/assets/6894585/8c3ec252-4a2b-40ab-b515-6510ce939556)
![imatge](https://github.com/TobikoData/sqlmesh/assets/6894585/b0d5a73c-c9d9-4222-a591-ea1a9a735fea)

**EDIT:** I've tried with an older version of sqlmesh (0.72.0) and UI works as expected, so it probably is something introduced recently. 

Thanks for checking it out!

**Comment by mykalmax** [comment]
@martsec
Do you see any errors in terminal ? 
did you try to hard reload browser / open in private mode / delete cache ?
here is a [link to deployed v0.79.0](https://sqlmesh-ui-basic.fly.dev/editor), same issues ?
Thanks

**Comment by martsec** [comment]
I don't see any errors. With the link you sent me, it's happening the same except in the `config.yaml` Odd

**Comment by mykalmax** [comment]
the issue is with Win|Mac path difference (`/` and `\`). Fixing it now.

**Comment by martsec** [comment]
Thank you!

**Comment by mykalmax** [comment]
@martsec  could you please try [deployed demo](https://sqlmesh-ui-basic.fly.dev/editor)

**Comment by mykalmax** [comment]
Could you please check if you still see issue [here](https://sqlmesh-ui-basic.fly.dev/editor)

**Comment by mykalmax** [comment]
Fixed in v0.81.0
